### PR TITLE
Update teacher.jade

### PR DESCRIPTION
**Pull request by popey456963**
Removed hour of code references.  Added array to things learnt.  Added minimum age.

### CODE REVIEW

File: app/locale/en.coffee [hunk 1]
```diff
@@ -546,8 +546,10 @@
     violent_2: "If you are comfortable having your students play Angry Birds, you will be comfortable with CodeCombat."
     for_girls_title: "Is it for girls?"
     for_girls_1: "There are three game modes in CodeCombat: building, puzzles, and combat. We have intentionally designed each to appeal to both boys and girls and think that the building and puzzle levels especially differentiate the game from violent triple A titles that repel female players."
+    age_requirement_title: "What age is it aimed at?"
+    age_requirement_1: "The recommend minimum age is 9, however children as young as 7 can play the first stages with the assistance of an adult."
```
Comment (Imperadeiro98) [inline app/locale/en.coffee]: Wouldn't it be recommended?

Comment (popey456963) [inline app/locale/en.coffee]: That's a good point, I'll update it now.

**Comment by Imperadeiro98** [comment]
And also in the string itself (sorry for not telling earlier).

**Comment by Imperadeiro98** [comment]
Thank you.

**Pull request merged**